spec/factories/activity.rb

```diff
@@ -384,6 +384,7 @@
   trait :level_form_state do
     form_state { "level" }
     level { nil }
+    source_fund_code { nil }
```

Comment:
As well as defining named constants for the fund codes, it would be useful to have a trait for creating Newton/GCRF activities that encapsulates how the fund relationship is represented.